certora/specs/CER-1-Owner/CER-59-Owner-override.spec

```diff
@@ -0,0 +1,24 @@
+// CER-59: EVC MUST NOT override the already stored Owner for a 
+// given addressPrefix
+import "../utils/IsMustRevertFunction.spec";
+
+persistent ghost bool overwroteOwner {
+    init_state axiom overwroteOwner == false;
+}
+
+hook Sstore currentContract.ownerLookup[KEY bytes19 address_prefix] address newValue (address oldValue) STORAGE {
+    if (oldValue != 0 && newValue != oldValue) {
+        overwroteOwner = true;
+    }
+}
+
+rule neverOverwriteOwner (method f) filtered {f ->
+    !isMustRevertFunction(f) &&
+    f.selector != sig:EthereumVaultConnectorHarness.permit(address,uint256,uint256,uint256,uint256,bytes,bytes).selector
```

Comment:
why do we need to filter out the `permit`?

Reply:
We don't need this.... I am really not sure why I had it, but deleting this filter just works, so it is deleted now. Apologies for this and great catch.